src/main/java/com/minecolonies/coremod/colony/ColonyManager.java

```diff
@@ -64,17 +64,23 @@
      * The list of all colonies.
      */
     @NotNull
-    private static final ColonyList<Colony>         colonies              = new ColonyList<>();
+    private static final ColonyList<Colony>         colonies        = new ColonyList<>();
     /**
      * The list of all colonies by world.
      */
     @NotNull
-    private static final Map<Integer, List<Colony>> coloniesByWorld       = new HashMap<>();
+    private static final Map<Integer, List<Colony>> coloniesByWorld = new HashMap<>();
     /**
      * The list of colony views.
      */
     @NotNull
-    private static final ColonyList<ColonyView>     colonyViews           = new ColonyList<>();
+    private static final ColonyList<ColonyView>     colonyViews     = new ColonyList<>();
+
+    /**
+     * A buffer value to be sure to be outside of the colony.
+     */
+    private static final int BUFFER                                 = 10;
```

Comment:
because players can configure it. What if the put the padding to 0.
I still want to be sure here.